server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/FileDescriptionNode.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree.variables;
+
+import lombok.Getter;
+import org.eclipse.lsp.cobol.core.model.Locality;
+
+/**
+ * Class for all File Description Entry (FD) or Sort File Description (SD) Entry variables. These
+ * variables are just logical.
+ */
+public class FileDescriptionNode extends VariableNode {
```

Comment:
Should contain also `@ToString(callSuper = true)` and `@EqualsAndHashCode(callSuper = true)`

Reply:
done